coordinator/src/app.ts

```diff
@@ -29,8 +30,19 @@ const init = async () => {
 
   server.route(routes)
 
+  await server.register({
+    plugin: HapiPino,
+    options: {
+      // For non-local environments, dont pretty print to avoid spamming logs
+      prettyPrint: process.env.ENVIRONMENT_NAME === "local",
+      // Redact Authorization headers, see https://getpino.io/#/docs/redaction
+      redact: ["req.headers.authorization"]
+    }
+  })
+
   await server.start()
-  console.log("Server running on %s", server.info.uri)
+  server.log("info", `Server running on ${server.info.uri}`)
+  server.logger
```

Comment:
?

Reply:
Assume this was left over from debugging but cba waiting for pipelines to run again